CONTRIBUTING.md

```diff
@@ -19,6 +19,74 @@ The Valkey project is led by a Technical Steering Committee, whose responsibilit
 * Want to help with documentation? [Move on to valkey-doc](https://github.com/valkey-io/valkey-doc)
 * Report a vulnerability? See [SECURITY.md](SECURITY.md)
 
+## How to Install clang-format 18 on a Debian Box
+
+```sh
+sudo apt-get update -y
+sudo apt-get upgrade -y
+sudo apt-get install software-properties-common -y
+wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor | sudo tee /usr/share/keyrings/llvm-toolchain.gpg > /dev/null
+echo "deb [signed-by=/usr/share/keyrings/llvm-toolchain.gpg] http://apt.llvm.org/$(lsb_release -cs)/ llvm-toolchain-$(lsb_release -cs)-18 main" | sudo tee /etc/apt/sources.list.d/llvm.list
+sudo apt-get update -y
+sudo apt-get install clang-format-18 -y
+```
+
+## How to Run clang-format-check Action Locally
+
+1. Get [`act`](https://github.com/nektos/act).
+2. Commit the changes locally:
+    ```sh
+    git commit -a -s
+    ```
```

Comment:
```suggestion
```

Instead, I think you should indicate it runs on what is current committed locally.